pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.greenbird.utilihive.stubs</groupId>
+    <artifactId>utilihive-backend-stub-sftp</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Utilihive SFTP Backend Stub</name>
+    <description>Stub for SFTP backend, for use in functional testing of Utilihive integration flows</description>
+    <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server/blob/main/LICENSE</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Greenbird Integration Technology</name>
+            <email>operations@greenbird.com</email>
```

Comment:
Is this address real? Maybe it should point to support instead?

Reply:
Changed to support